.vscode/tasks.json

```diff
@@ -0,0 +1,10 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+      {
+        "label": "bootstrap",
+        "type": "shell",
+        "command": "${workspaceFolder}/test_app/scripts/bootstrap.sh startpostgres",
```

Comment:
So this is like a separate button you click in VSCode? I get why you need to give it your python interpreter and have it wrap it or whatever to do it in debug mode. But it seems a lot less clear how to manage a background container.

Reply:
Maybe it would be easier to just let VSCode use sqlite3. We were until like this morning.